devices/Bq2579x/Bq2579x.cs

```diff
@@ -261,6 +268,58 @@ public class Bq2579x : IDisposable
         [Property]
         public ThermalShutdownThreshold ThermalShutdownThreshold { get => GetThermalShutdownThreshold(); set => SetThermalShutdownThreshold(value); }
 
+        /// <summary>
+        /// Gets or sets charge voltage for JEITA high temperature range (TWARN � THOT). 
```

Comment:
non ascii character that you should remove/replace:
![image](https://github.com/nanoframework/nanoFramework.IoT.Device/assets/8145578/11fc360c-a680-4376-bd60-75e2ede8f49e)

Reply:
copy paste from website...